src/test/kotlin/racingcar/domain/RaceResultTest.kt

```diff
@@ -0,0 +1,41 @@
+package racingcar.domain
+
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.style.DescribeSpec
+import io.kotest.matchers.shouldBe
+import racingcar.domain.exception.CannotFindWinnerException
+
+class RaceResultTest : DescribeSpec({
+    describe("레이싱 자동차 게임 결과") {
+        it("마지막 라운드의 단일 우승자를 찾을 수 있다.") {
+            val firstRoundHistory = RoundHistory(1, listOf(Pair("pobi", 1), Pair("crong", 1)))
+            val secondRoundHistory = RoundHistory(1, listOf(Pair("pobi", 2), Pair("crong", 1)))
```

Comment:
2라운드니까 ReounHistory의 `round` 값이 **2**여야하지 않나요? 🤔
```suggestion
            val secondRoundHistory = RoundHistory(2, listOf(Pair("pobi", 2), Pair("crong", 1)))
```